CMakeLists.txt

```diff
@@ -13,6 +13,14 @@ elseif (MSVC)
 	endif()
 endif()
 
+# Python Bindings
+find_package(SWIG 3.0.8 REQUIRED)
```

Comment:
Can we make SWIG/Python/etc optional, so if I just want to build the C++ stuff I don't need to get swig setup.